### DISCUSSION TITLE
How do you set inc-rename.nvim?

### DISCUSSION THREAD

**Discussion by Demianeen** [LazyVim/LazyVim#2653] [Q&A] [answered]
I found that lazyvim checks for util in lazyvim [repo folder](https://github.com/LazyVim/LazyVim/blob/91126b9896bebcea9a21bce43be4e613e7607164/lua/lazyvim/plugins/lsp/keymaps.lua#L42) before enabling the inc-rename nvim. But as far as I know, the Lazyvim repo will be overwritten after each update. I haven't found any discussions on this, so this is probably obvious. 

Do I don't understand something?😅

**Comment by dpetka2001** [comment, marked as answer]
That line doesn't check for util. It actually requires the module `lazyvim.util` and calls the function `has()` that is defined in that module, which checks **if** `inc-rename.nvim` is installed.

**Reply by Demianeen** [reply]
Oh, wow! Does lazyvim somewhere describes all of its hidden features like inc-rename.nvim? Or is it more just to explore the repo by yourself?

**Reply by dpetka2001** [reply]
`inc-rename.nvim` isn't installed by default in LazyVim. The maintainer might have put that in the keymaps because he might be using it in his personal configuration, so that if any user would install it, it might be available. All the default installed plugins can be found in the [docs](https://www.lazyvim.org/plugins) under each sub-section (coding, editor, etc).
